lib/src/navigation/routes/bindings_interface.dart

```diff
@@ -1,7 +1,34 @@
+/// [Bindings] should be extended or implemented.
+/// When using [GetMaterialApp], all [GetPage]s and navigation methods (like Get.to())
+/// have a [binding] property that takes an instance of Bindings to manage the
+/// dependencies() (via [Get.put()]) for the Route you are opening.
 abstract class Bindings {
   void dependencies();
 }
 
+/// Simplifies Bindings generation from a single callback.
+/// To avoid the creation of a custom Binding instance per route.
+///
+/// Example:
+/// ```
+/// GetPage(
+///   name: '/',
+///   page: () => Home(),
+///   binding: BindingsBuilder(() => Get.put(HomeController())),
```

Comment:
i absolutely love that!
i was thought that having a separate file for the bindings was kinda confusing, and this will help a lot in code readability (at least for me)
thanks for that ❤️

Reply:
Glad u like it, i had the same feeling about Bindings class

Reply:
Nice one @roipeker , sometimes it's a little bit over if we must create a binding file just for a controller.

Thanks